client/src/app/patient-summary/patient-summary.component.html

```diff
@@ -5,7 +5,7 @@
         id="aud-button" 
         (click)="audioLoginClick()" 
         type="button" 
-        class="btn btn-primary btn-sm">Audiologist Log In</button>
+        class="btn btn-primary btn-sm">Log In</button>
```

Comment:
Actually, I think we should rename this button to “Submit Survey” or “Start Tests”. Anyone else have thoughts?

Reply:
Start Tests sounds better than Submit Survey, and I do think that it makes more sense than Log In in that page. I think its a good idea.